RecoEcal/Configuration/python/RecoEcal_EventContent_cff.py

```diff
@@ -107,11 +41,54 @@
 from Configuration.Eras.Modifier_ppRef_2017_cff import ppRef_2017
 #HI-specific products needed in pp scenario special configurations
 for e in [pA_2016, peripheralPbPb, pp_on_AA_2018, pp_on_XeXe_2017, ppRef_2017]:
-    for ec in [RecoEcalAOD.outputCommands, RecoEcalRECO.outputCommands, RecoEcalFEVT.outputCommands]:
-        e.toModify( ec, func=lambda outputCommands: outputCommands.extend(['keep recoSuperClusters_correctedIslandBarrelSuperClusters_*_*',
-                                                                           'keep recoSuperClusters_correctedIslandEndcapSuperClusters_*_*'
-                                                                           ])
-                    )
-    for ec in [RecoEcalRECO.outputCommands, RecoEcalFEVT.outputCommands]:
-        e.toModify( ec, func=lambda outputCommands: outputCommands.extend(['keep recoCaloClusters_islandBasicClusters_*_*'])
-                    )
+    e.toModify( RecoEcalAOD.outputCommands, 
+                func=lambda outputCommands: outputCommands.extend(['keep recoSuperClusters_correctedIslandBarrelSuperClusters_*_*',
+                                                                   'keep recoSuperClusters_correctedIslandEndcapSuperClusters_*_*'])
+              )
+# RECO content
+RecoEcalRECO = cms.PSet(
+    outputCommands = cms.untracked.vstring(
+	# Barrel clusters
+        'keep *_hybridSuperClusters_*_*',
+        'keep recoSuperClusters_correctedHybridSuperClusters_*_*',
+	# Endcap clusters
+        'keep *_multi5x5SuperClusters_*_*',
+        'keep recoSuperClusters_multi5x5SuperClusters_*_*',
+        'keep recoSuperClusters_multi5x5SuperClustersWithPreshower_*_*',
+        # Particle Flow superclusters
+        'keep *_particleFlowSuperClusterECAL_*_*',
+        'keep *_particleFlowSuperClusterOOTECAL_*_*',
+	# DROP statements # not used
```

Comment:
How did you verify that these drop statements are not used?
@afiqaize @SohamBhattacharya

Reply:
By my event content comparison check, these drop statements don't really act on the RECO/FEVT event content. When I checked every event content without these drop statements like this modified file, there is no difference in event contents.
@afiqaize @SohamBhattacharya Please confirm if these 4 drop statements can be removed.

Reply:
For now, it would be good to have these drop statements in RecoEcalRECO PSets as the original for now. I'll not remove them. It will safe in the new commit.